docs/tutorials/infrastructure/kne/srl-with-kne/srl-with-oc-services.md

```diff
@@ -0,0 +1,314 @@
+    }
+    ``` -->
+
+gNMI instance configured in the `mgmt` network instance uses native [YANG models](../../../../yang/yang.md). This is driven by the default configuration value of the `/system/gnmi-server/network-instance[name=mgmt]/yang-models` leaf and selects which models are going to be used when gNMI paths are provided without the [`origin`](https://github.com/openconfig/reference/blob/c243b35b36e366852f9476c87fb2efe6e9050dfe/rpc/gnmi/gnmi-specification.md#222-paths) information in the path.
+
+Users can change the `yang-models` leaf value to `openconfig` should they want to use openconfig paths with gNMI without providing the `origin` value.
+
+??? "Example"
+    gNMI service can be tested using [gnmic](https://gnmic.kmrd.dev) cli client.
+    === "Capabilities"
+        ```bash
+        ❯ gnmic -a 172.18.0.50:9339 -u admin -p admin --skip-verify capabilities
+        gNMI version: 0.7.0
+        supported models:
+          - urn:srl_nokia/aaa:srl_nokia-aaa, Nokia, 2022-06-30
+          - urn:srl_nokia/aaa-password:srl_nokia-aaa-password, Nokia, 2022-06-30
+          - urn:srl_nokia/aaa-types:srl_nokia-aaa-types, Nokia, 2021-11-30
+          - urn:srl_nokia/acl:srl_nokia-acl, Nokia, 2022-06-30
+        -- snip --
+        ```
+    === "Get using native YANG models"
+        By default, native YANG models are used by the gNMI server. This means that paths without the `origin` information are assumed to belong to the native YANG models.
+        ```bash
+        ❯ gnmic -a 172.18.0.50:9339 -u admin -p admin --skip-verify -e JSON_IETF \
+          get --path /system/information/version
+        [
+          {
+            "source": "172.18.0.50:9339",
+            "timestamp": 1665490620272174602,
+            "time": "2022-10-11T14:17:00.272174602+02:00",
+            "updates": [
+              {
+                "Path": "srl_nokia-system:system/srl_nokia-system-info:information/version",
+                "values": {
+                  "srl_nokia-system:system/srl_nokia-system-info:information/version": "v22.6.4-90-g4b19af2d95"
+                }
+              }
+            ]
+          }
+        ]
+        ```
+    === "Get using Openconfig YANG models"
+        When openconfig is not configured to be a default schema for gNMI server, users need to set the `origin` field of the path prefix to `openconfig` value:
+        ```bash
+        ❯ gnmic -a 172.18.0.50:9339 -u admin -p admin --skip-verify -e JSON_IETF \
+          get --prefix "openconfig:/" --path "/system/state/hostname"
+        [
+          {
+            "source": "172.18.0.50:9339",
+            "timestamp": 1665493415999114602,
+            "time": "2022-10-11T15:03:35.999114602+02:00",
+            "updates": [
+              {
+                "Path": "openconfig-system:system/state/hostname",
+                "values": {
+                  "openconfig-system:system/state/hostname": "srl1"
+                }
+              }
+            ]
+          }
+        ]
```

Comment:
@alexmasi 
additionally I've added an example on how to test OC paths with gnmi.
Caveat here is to in using path prefixes with origin